src/components/widgets/filesystem/setupMonaco.ts

```diff
@@ -1,5 +1,27 @@
+import 'monaco-editor/esm/vs/editor/editor.all.js'
+
+import 'monaco-editor/esm/vs/editor/standalone/browser/accessibilityHelp/accessibilityHelp.js'
+import 'monaco-editor/esm/vs/editor/standalone/browser/iPadShowKeyboard/iPadShowKeyboard.js'
+// import 'monaco-editor/esm/vs/editor/standalone/browser/inspectTokens/inspectTokens.js'
+// import 'monaco-editor/esm/vs/editor/standalone/browser/quickAccess/standaloneHelpQuickAccess.js'
+import 'monaco-editor/esm/vs/editor/standalone/browser/quickAccess/standaloneGotoLineQuickAccess.js'
+// import 'monaco-editor/esm/vs/editor/standalone/browser/quickAccess/standaloneGotoSymbolQuickAccess.js'
+import 'monaco-editor/esm/vs/editor/standalone/browser/quickAccess/standaloneCommandsQuickAccess.js'
+// import 'monaco-editor/esm/vs/editor/standalone/browser/referenceSearch/standaloneReferenceSearch.js'
+
+// import 'monaco-editor/esm/vs/language/typescript/monaco.contribution'
+import 'monaco-editor/esm/vs/language/css/monaco.contribution'
+import 'monaco-editor/esm/vs/language/json/monaco.contribution'
+// import 'monaco-editor/esm/vs/language/html/monaco.contribution'
+// import 'monaco-editor/esm/vs/basic-languages/monaco.contribution'
```

Comment:
Are the commented out imports here intended?

Reply:
I left those as reference, but thinking again, maybe I'll just add a link to the list of possible imports instead.

Reply:
All clear now!